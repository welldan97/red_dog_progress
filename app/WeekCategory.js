import React from 'react';
import { Table } from 'reactstrap';

const WeekCategory = ({ today,category }) => {
  const cellClassName = (day) =>
    day === today.getDay() ? 'table-info' : '';
  return(
    <tr>
    <th scope="row" > {category.name} </th>
    <td  className={cellClassName(1)}> 12 hours</td>
    <td className={cellClassName(2)}> 5 hours</td>
    <td  className={cellClassName(3)} />
    <td  className={cellClassName(4)} />
    <td  className={cellClassName(5)} />
    <td  className={cellClassName(6)} />
    </tr>
  );
}
export default WeekCategory;
