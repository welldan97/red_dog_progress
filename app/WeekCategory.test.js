import React from 'react';
import renderer from 'react-test-renderer';

import WeekCategory from './WeekCategory';
import db from '../fixtures/db.json';

it('It renders', () => {
  const attributes = {
    today: new Date('2017-08-18T03:24:00'),
    category: db.categories[0],
  };
  const component = renderer.create(<WeekCategory {...attributes} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
