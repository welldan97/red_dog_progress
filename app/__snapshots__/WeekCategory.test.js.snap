// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders 1`] = `
<tr>
  <th
    scope="row"
  >
     
    Meditation
     
  </th>
  <td
    className=""
  >
     12 hours
  </td>
  <td
    className=""
  >
     5 hours
  </td>
  <td
    className=""
  />
  <td
    className=""
  />
  <td
    className="table-info"
  />
  <td
    className=""
  />
</tr>
`;
